tests/molecule/resources/verify.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: Verify
+  hosts: all
+  gather_facts: false
+  tasks:
+    - name: Verify firewalld
+      ansible.builtin.include_tasks:
+        file: verify-firewalld.yml
+        apply:
+          tags:
+            - firewalld
```

Comment:
apply the tag `firewalld` to all tasks within the `verify-firewalld.yml` playbook, otherwise the tasks won't run